activiti-core/activiti-engine/src/main/java/org/activiti/engine/impl/bpmn/helper/ErrorPropagation.java

```diff
@@ -60,7 +60,7 @@ public static void propagateError(String errorRef, DelegateExecution execution)
             isCatchExecutedForProcess = executeCatch(eventMap, execution, errorRef);
         }
 
-        if (isCallActivity(execution)) {
+        if (isCallActivity(execution) && !isCatchExecutedForProcess) {
```

Comment:
We don't need to call `isCallActivity` if `isCatchExecutedForProcess` is `true`

```suggestion
        if (!isCatchExecutedForProcess && isCallActivity(execution)) {
```